server/main.py

```diff
@@ -60,14 +60,15 @@ async def calculate_spectrum(payload: Payload):
             databank=payload.database,
             use_cached=True,
         )
+        if simulate_slit is not None:
```

Comment:
Since we're already passing the value, we could do something like `if useSimulateSlit:`